### PR TITLE
Create new Ink app in a new directory

### LINKED ISSUE #1: Should create in named folder, not current folder
create-react-app (and most other generators) will require a folder to generate into so you don't accidentally fill your root directory with a bunch of files.  Should be `npx create-ink-app my-app` to create the app at the directory `my-app`

### PR DESCRIPTION
**Pull request by kevin940726**
Close #1.

Added support to pass a new input `<project-directory>` indicating where the project should be scaffolded in. Currently the input is optional, the current directory will be the default if it's not present, so that we can make this feature backward-compatible. In the future, we should at least add a warning if the input is not passed.

I also added some basic helpful message upon successfully created an app. Wordings are TBD.

**Example**:

```bash
create-ink-app # Create an Ink app in the current working directory
create-ink-app . # Same as above
create-ink-app my-cli # Create an Ink app in the "my-cli" directory relative to the current working directory
create-ink-app --typescript my-cli # Typescript template also works
```

I intentionally move some logic to `cli.js` so that in the future we can support programmatic API by directly importing `index.js` as well.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -7,11 +7,9 @@
 This helper tool scaffolds out basic project structure for Ink apps and lets you avoid the boilerplate and get to building beautiful CLIs in no time.
 
 ```bash
-$ mkdir my-fancy-cli
-$ cd my-fancy-cli
-$ npx create-ink-app
+$ npx create-ink-app my-fancy-cli
 # Or create with TypeScript React
-$ npx create-ink-app --typescript
+$ npx create-ink-app --typescript my-fancy-ts-cli
```
Comment (BrunoS3D) [inline readme.md:12]:
```suggestion
$ npx create-ink-app my-fancy-ts-cli --typescript
```

**Review by BrunoS3D — APPROVED** [review]
Nice job :rocket:

**Comment by BrunoS3D** [comment]
What do you think of a `--template` flag like in [create-react-app](https://create-react-app.dev/docs/adding-typescript/#installation)?

It would be interesting if the value received in the template argument matches the _name of the template_ folder in the `templates` directory of this project -> 

## Example:

```bash
npx create-ink-app my-fancy-ts-cli --template typescript-eslint
```

The [templates](https://github.com/vadimdemedes/create-ink-app/tree/master/templates) folder
```bash
# ...
templates
 |- _common
 |- js
 |- ts # rename to "typescript"
 |- typescript-eslint
   |- # ...
   |- tsconfig.json
   |- .eslintrc
   |- # ...
# ...
```

File: cli.js [hunk 2]
```diff
@@ -1,19 +1,46 @@
...
-	console.error(error.stack);
-	process.exit(1);
-});
+	Examples
+		$ create-ink-app my-cli
+		$ create-ink-app .
+`,
+	{
+		flags: {
+			typescript: {
```
Comment (mattferrin) [inline cli.js:21]: I would recommend making "project-directory" a flag too. It seems safer than accessing it as index 0. 

`cli.input[0]` seems fragile if for some reason something is misaligned and at position `cli.input[1]` instead.

**Review by mattferrin — COMMENTED** [review]
I didn't test anything. I left 1 suggestion. I didn't notice anything concerning that should prevent merging.

File: cli.js (hunk 2 above)
Comment (mattferrin) [reply cli.js:21]: (I don't feel strongly about this.)

Comment (mattferrin) [reply cli.js:21]: I retract my suggestion. Convenience is more important.

**Comment by vadimdemedes** [comment]
Thanks @kevin940726 👍

**Pull request merged**